apps/studio/src/features/dashboard/components/PageSettingsModal.tsx

```diff
@@ -121,8 +121,8 @@ const PageSettingsModalContent = ({
       await utils.collection.invalidate()
 
       toast({
-        title: "Saved page settings",
-        description: "Publish this page for your changes to go live.",
+        title: "Saved and published page settings",
+        description: "Check your site later to see the changes",
```

Comment:
What would "later" mean to a user?

Might want to check the copy with @sehyunidaaa

Reply:
I think "few mins" works

Avg case scenario, it should not take more than 10 mins. 
Worst case scenario is 15min

Reply:
Lets just use the original wording? 

`description: "Check your site in 5-10 minutes to view it live."`

Reply:
lgtm